src/setup-dotnet.ts

```diff
@@ -110,9 +110,27 @@ function getVersionFromGlobalJson(globalJsonPath: string): string {
   if (globalJson.sdk && globalJson.sdk.version) {
     version = globalJson.sdk.version;
     const rollForward = globalJson.sdk.rollForward;
-    if (rollForward && rollForward === 'latestFeature') {
-      const [major, minor] = version.split('.');
-      version = `${major}.${minor}`;
+    if (rollForward && rollForward.startsWith('latest')) {
+      const [major, minor, featurePatch] = (version || '').split('.');
+      const feature = featurePatch.substring(0, 1);
+      
+      switch (rollForward) {
+        case 'latestMajor':
+          version = '';
+          break;
+          
+        case 'latestMinor':
+          version = `${major}`;
+          break;
+          
+        case 'latestFeature':
+          version = `${major}.${minor}`;
```

Comment:
```suggestion
          version = `${major}.${minor}.x`;
```

Reply:
This is fine as a cosmetic change, but the action handled the previous format as well, unlike what was used for `latestPatch`.